### PR TITLE
Vendor package:tar and package:chunked_stream

### PR DESCRIPTION
**Pull request by sigurdm**

### CODE REVIEW

File: lib/src/third_party/chunked_stream/METADATA [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    "Utilities for working with chunked streams, such as byte streams which is"
+    "often given as a stream of byte chunks with type `Stream<List<int>>`."
+
+third_party {
+  url {
+    type: GIT
+    value: "https://github.com/google/dart-neats/"
+  }
+  version: "1.4.0"
+  last_upgrade_date { year: 2014 month: 10 day: 20 }
```
Comment (jonasfj) [inline lib/src/third_party/chunked_stream/METADATA:12]: Should we remove this?

File: lib/src/third_party/tar/.github/workflows/dart.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
+name: build
```
Comment (jonasfj) [inline lib/src/third_party/tar/.github/workflows/dart.yml:1]: I think we can skip this file...

**Review by jonasfj — APPROVED** [review]

File: lib/src/third_party/chunked_stream/METADATA (hunk 1 above)
Comment (sigurdm) [reply lib/src/third_party/chunked_stream/METADATA:12]: Or update it :)

File: lib/src/third_party/tar/.github/workflows/dart.yml (hunk 2 above)
Comment (sigurdm) [reply lib/src/third_party/tar/.github/workflows/dart.yml:1]: ++

**Pull request merged**